Lib/test/test_mmap.py

```diff
@@ -721,6 +721,8 @@ def test_resize_past_pos(self):
         self.assertRaises(ValueError, m.write_byte, 42)
         self.assertRaises(ValueError, m.write, b'abc')
 
+    # TODO: RUSTPYTHON
+    @unittest.skip
```

Comment:
```suggestion
    @unittest.skip("TODO: RUSTPYTHON, WhateverError: this thingy doesn't work")
```

…but please replace the `WhateverError: this thingy doesn't work` part with something more useful than that.

Reply:
Let me change it to the following
```
@unittest.skip('TODO: RUSTPYTHON, Number Protocol is not properly implemented yet')
```